### PR TITLE
Instance admin async get instance

### PR DESCRIPTION
**Pull request by roopak-qlogic**
Issue Fixex#1524

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-cpp/1724)

### CODE REVIEW

File: google/cloud/bigtable/instance_admin.h [hunk 1]
```diff
@@ -238,6 +266,35 @@ class InstanceAdmin {
...
+   * Sends an asynchronous request to get information about existing cluster of
+   * an instance.
+   *
+   * @warning This is an early version of the asynchronous APIs for Cloud
+   *     Bigtable. These APIs might be changed in backward-incompatible ways. It
+   *     is not subject to any SLA or deprecation policy.
+   *
+   * @param instance_id the id of the instance in the project.
+   * @param cluster_id the id of the cluster in the project that needs to be
+   * reterived.
```
Comment (coryan) [inline google/cloud/bigtable/instance_admin.h]: nit: spelling of **retrieved**.

File: google/cloud/bigtable/examples/run_examples_utils.sh [hunk 2]
```diff
@@ -148,6 +148,45 @@ function run_all_instance_admin_examples {
...
+
+  # Create a (very likely unique) instance name.
+  local -r INSTANCE="in-$(date +%s)"
+
+  run_example ./bigtable_samples_instance_admin create-instance \
+      "${project_id}" "${INSTANCE}" "${zone_id}"
+  run_example ./instance_admin_async_snippets async-get-instance \
+      "${project_id}" "${INSTANCE}"
+  run_example ./bigtable_samples_instance_admin create-cluster \
+      "${project_id}" "${INSTANCE}" "${INSTANCE}-c2" "us-central1-a"
```
Comment (coryan) [inline google/cloud/bigtable/examples/run_examples_utils.sh]: Please add a comment like:

```sh
# TODO(#1726) - do not use a hard-coded zone id here.
```

File: google/cloud/bigtable/instance_admin.h (hunk 1 above)
Comment (roopak-qlogic) [reply google/cloud/bigtable/instance_admin.h]: Fixed

File: google/cloud/bigtable/examples/run_examples_utils.sh (hunk 2 above)
Comment (roopak-qlogic) [reply google/cloud/bigtable/examples/run_examples_utils.sh]: Updated

**Pull request merged**